components/server/ee/src/workspace/gitpod-server-impl.ts

```diff
@@ -2161,6 +2161,20 @@ export class GitpodServerEEImpl extends GitpodServerImpl {
         return undefined;
     }
 
+    async getUsageLimitForUser(ctx: TraceContext): Promise<number | undefined> {
```

Comment:
@andrew-farries Why can't we generalize this to `getUsageLimit(attributionId)`? :thinking:  I think that way it would be way less code and complexity. And there should be zero differences between the two, as access is abstracted away behind the `guardCostCenterAccess`.

Reply:
Do you mean keep `(get|set)UsageLimitFor(Team|User)` on the server API but have them call a new method `getUsageLimitFor(attributionId)`? 

Or remove the `get|set` methods from the server in favour of a new `getUsageLimitFor(attributionId)`?

Reply:
Ultimately the API should have:
 - getUsageLimit(attributionId)
 - setUsageLimit(attributionId)

If it's easier to 1. introduce the new methods, and remove the old usage+methods in a 2. PR that's fine with me.

Reply:
Rebased to add the two new methods, and rewrote the PR title + description.

We'll leave the old `get|setUsageLimitForTeam` methods and update the dashboard to use the more general ones in another PR.